HandleNHNotificationCallActivity/__tests__/handler.test.ts

```diff
@@ -54,11 +60,50 @@ const aDeleteInStalltionMessage: DeleteInstallationMessage = {
   kind: "DeleteInstallation" as any
 };
 
+const aNHConfig = {
+  AZURE_NH_ENDPOINT: envConfig.AZURE_NH_ENDPOINT,
+  AZURE_NH_HUB_NAME: envConfig.AZURE_NH_HUB_NAME
+} as NotificationHubConfig;
```

Comment:
It's ok, but a more robust solution would be to decode:

```suggestion
const aNHConfig = NotificationHubConfig.decode({
  AZURE_NH_ENDPOINT: envConfig.AZURE_NH_ENDPOINT,
  AZURE_NH_HUB_NAME: envConfig.AZURE_NH_HUB_NAME
}).getOrElse(() => { 
  throw new Error(`Cannot decode aNHConfig`);
});

Reply:
Not sure if it's really useful since this is a test, but ok, I'll do this using `getorElseL`

Reply:
Done in [2eadf67](https://github.com/pagopa/io-functions-pushnotifications/pull/4/commits/2eadf6746ab882b6b39c85ff15d64f1a3b9057fb)